cupy/fft/fft.py

```diff
@@ -114,7 +114,10 @@ def _fft_c2c(a, direction, norm, axes, overwrite_x):
     return a
 
 
-def _fft(a, s, axes, norm, direction, value_type='C2C', overwrite_x=False):
+def _fft(a, s, axes, norm, direction, value_type='C2C',
+         overwrite_x=False, **kwargs):
+    kwargs=None # guard
```

Comment:
I would just add `plan=None` instead of `**kwargs` to the function signature and then set:

```python
if plan is not None:
    raise NotImplementedError("Use of a pre-existing plan is not currently supported for 1D transforms.")
```

Reply:
Make sense. I hate `**kwargs` anyway. Done.